icons/shield-close.svg

```diff
@@ -0,0 +1,15 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M12 22s8-4 8-10V5l-8-3-8 3v7c0 6 8 10 8 10z" />
+  <path d="M15 9l-6 6" />
+  <path d="M9 9l6 6" />
```

Comment:
I think the x looks a bit better with a smaller one. 
```suggestion
  <line x1="9.5" y1="9" x2="14.5" y2="14" />
  <line x1="14.5" y1="9" x2="9.5" y2="14" />
```
![image](https://user-images.githubusercontent.com/11825403/106940307-b00a9f80-6721-11eb-8523-9168c16d784a.png)

P.S. Can you rename this icon to `shield-x`?